build.yaml

```diff
@@ -674,6 +689,13 @@ libs:
   - grpc
   - gpr_test_util
   - gpr
+- name: codegen_lib
+  build: all
```

Comment:
This means this becomes installable, which is probably not desirable. Private maybe ?